src/Network/Network/FirewallPolicy/PolicySettings/AzureApplicationGatewayFirewallPolicySetting.cs

```diff
@@ -33,7 +33,15 @@ public class AzureApplicationGatewayFirewallPolicySetting : NetworkBaseCmdlet
         [ValidateSet("Disabled", "Enabled", IgnoreCase = true)]
         [ValidateNotNullOrEmpty]
         public string State { get; set; }
-        
+
+        [Parameter(Mandatory = false, HelpMessage = "Whether allow WAF to enforce request body limits.")]
+        [ValidateNotNullOrEmpty]
+        public bool RequestBodyEnforcement { get; set; }
```

Comment:
Should we rather change to EnableRequestBodyEnforcement to make it more clear?

Reply:
This name is from swagger

Reply:
Actually we cannot change the name of the variable because we need it to match the one in the swagger network sdk, else the test would fail.